core/controllers/reader.py

```diff
@@ -364,60 +306,25 @@ def post(self, exploration_id):
         Args:
             exploration_id: str. The ID of the exploration.
         """
-        playthrough_data = self.payload.get('playthrough_data')
         try:
-            unused_playthrough = stats_domain.Playthrough.from_dict(
-                playthrough_data)
-        except utils.ValidationError as e:
-            raise self.InvalidInputException(e)
-
-        try:
-            self.current_issue_schema_version = self.payload[
-                'issue_schema_version']
+            issue_schema_version = self.payload['issue_schema_version']
```

Comment:
I believe our typical practice is to do self.payload.get() and then do "if x is None, raise".